src/sentry_plugins/segment/plugin.py

```diff
@@ -0,0 +1,94 @@
+from __future__ import absolute_import
+
+from sentry import http
+from sentry.app import ratelimiter
+from sentry.plugins.base import Plugin
+from sentry.utils.hashlib import md5_text
+
+from sentry_plugins.base import CorePluginMixin
+
+
+class SegmentPlugin(CorePluginMixin, Plugin):
+    title = 'Segment'
+    slug = 'segment'
+    description = 'Send Sentry events into Segment.'
+    conf_key = 'segment'
+
+    endpoint = 'https://api.segment.io/v1/track'
+
+    def get_plugin_type(self):
+        return 'data-forwarding'
+
+    def get_config(self, project, **kwargs):
+        return [{
+            'name': 'write_key',
+            'label': 'Write Key',
+            'type': 'secret',
+            'required': True,
+        }]
+
+    def get_event_props(self, event):
+        props = {
+            'eventId': event.event_id,
+            'transaction': event.get_tag('transaction') or '',
+            'release': event.get_tag('sentry:release'),
+        }
+        if 'sentry.interfaces.Http' in event.interfaces:
+            http = event.interfaces['sentry.interfaces.Http']
+            headers = http.headers
+            if not isinstance(headers, dict):
+                headers = dict(headers or ())
+
+            props.update({
+                'requestUrl': http.url,
```

Comment:
@sperand-io are you already doing any kind of standard naming on these HTTP properties?